src/main/java/mate/academy/internetshop/controllers/GetAllOrdersController.java

```diff
@@ -9,16 +9,19 @@
 import mate.academy.internetshop.lib.Injector;
 import mate.academy.internetshop.model.Order;
 import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.UserService;
 
 public class GetAllOrdersController extends HttpServlet {
```

Comment:
```suggestion
public class GetUserOrdersController extends HttpServlet {
```
думаю краще так назвати, адже можна подумати, що тут ми витягуємо всі замовлення з бази а не замовлення конкретного юзера

Reply:
Done.